sound/soc/sof/ipc4-topology.c

```diff
@@ -1035,6 +1036,18 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 			/* at this point amount of dwords */
 			copier_data->gtw_cfg.config_length >>= 2;
 			break;
+		case SOF_DAI_INTEL_ALH:
+		{
+			struct sof_ipc4_alh_configuration_blob *blob;
+
+			blob = devm_kzalloc(sdev->dev, sizeof(*blob), GFP_KERNEL);
```

Comment:
can we use kzalloc() and free in unprepare()?

Reply:
> can we use kzalloc() and free in unprepare()?

In fact, blob is useless after 
```
memcpy(*ipc_config_data + sizeof(*copier_data),
                       *data, copier_data->gtw_cfg.config_length * 4);
```
So I will free after it.